hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -73,10 +73,18 @@ def pulp_enabled?
     module_enabled?('foreman_proxy_plugin_pulp') && (param_value('foreman_proxy_plugin_pulp', 'pulpnode_enabled') || param_value('foreman_proxy_plugin_pulp', 'enabled'))
   end
 
+  def pulp_present?
```

Comment:
It's actually identical to https://github.com/theforeman/foreman-installer/pull/540/files#diff-c6fdf0194de166ebab0dde67cd968490R84 but in the interest of having a consistent API I defined both.

The alternative would be relying on `pulpcore_present?` at https://github.com/theforeman/foreman-installer/pull/540/files#diff-d063850194f4a36e3364bdee036f452fR1 which doesn't feel correct even if it would accomplish the same thing while saving a few lines of codes that we are planning to remove in the not so distant future anyway.